File: Data Structures & Algorithms/concatenation-of-array/submission-1.cpp
class Solution {
public:
    vector<int> getConcatenation(vector<int>& nums) {
        std::cout << nums.size();
        std::vector<int> ans(nums.size()*2);
        std::copy(nums.begin(), nums.end(), ans.begin());
        std::copy(nums.begin(), nums.end(), ans.begin() + nums.size());
        return ans;
    }
}
;

File: Data Structures & Algorithms/duplicate-integer/submission-1.java

class Solution {
    public boolean hasDuplicate(int[] nums) {
        HashSet<Integer> nummap = new HashSet<>();
        for(int i = 0; i < nums.length; i++) {
            if(!nummap.contains(nums[i])){
                nummap.add(nums[i]); 
            } else {
                return true;
            } 
        }
        return false;
    }
}